authorization/v1alpha1/register.go

```diff
@@ -0,0 +1,40 @@
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	GroupName     = "authorization.openshift.io"
+	GroupVersion  = schema.GroupVersion{Group: GroupName, Version: "v1alpha1"}
```

Comment:
in the group level install, set the prioritized versions to v1 then v1alpha1

Reply:
Fixed.